src/components/Titlebar.vue

```diff
@@ -25,19 +31,59 @@
 
 <script lang="ts">
 import { Component, Vue } from 'vue-property-decorator';
+import { Getter, Mutation } from 'vuex-class';
+import { EditorModels } from '@/store/editors/types';
+import { download } from '@/file/Utils';
+import { FILENAME, saveEditor, saveEditors } from '@/file/EditorAsJson';
 
 @Component
 export default class Titlebar extends Vue {
-  share() {
+  @Getter('allEditorModels') editorModels!: EditorModels;
+  @Mutation('loadEditors') loadEditors!: (editorModels: EditorModels) => void;
+
+  private share() {
     console.log(`Share button pressed. ${this.$data}`);
   }
 
-  open() {
-    console.log(`Open button pressed. ${this.$data}`);
+  // eslint-disable-next-line class-methods-use-this
```

Comment:
I think something like this would be preferable here, instead of a class-method:
```
  private uploadfile = () => {
    const element = document.getElementById('upload-file');
    if (!element) return;
    element.click();
  };
```
Then at least we don't have the ESLint problem. Or we can change the lint file.

Reply:
Ah I didn't realise that is a thing, will push a change